### PR TITLE
fix histogram yaml awaitsFix problem

### PR DESCRIPTION
**Pull request by weizijun**
fix histogram with time series mappings tests problem.
remove index.mode=time_series

### CODE REVIEW

**Comment by weizijun** [comment]
@csoulios I open the new PR to replace https://github.com/elastic/elasticsearch/pull/82467

File: x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/analytics/histogram.yml [hunk 1]
```diff
@@ -175,17 +175,15 @@ setup:
 ---
 histogram with time series mappings:
   - skip:
-      version: "all"
-      reason: "AwaitsFix https://github.com/elastic/elasticsearch/issues/78443"
-
+      version: " - 7.15.99"
```
Comment (csoulios) [inline x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/analytics/histogram.yml:178]: I think this should be `7.16.99`

File: x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/analytics/histogram.yml [hunk 2]
```diff
@@ -175,17 +175,15 @@ setup:
...
+      reason: introduced in 7.16.0
```
Comment (csoulios) [inline x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/analytics/histogram.yml:179]: And this should probably be `7.17.0`

**Comment by csoulios** [comment]
@elasticmachine test this please

**Comment by csoulios** [comment]
@elasticmachine update branch

**Comment by csoulios** [comment]
@elasticmachine ok to test

**Review by csoulios — APPROVED** [review]
LGTM!

**Pull request merged**